### PR TITLE
Fix Matrix3/4 `makeTranslation` types

### PR DESCRIPTION
**Pull request by puxiao**
https://threejs.org/docs/index.html#api/en/math/Matrix4.makeTranslation

```
makeTranslation( x, y, z ) {

		if ( x.isVector3 ) {

			this.set(
				1, 0, 0, x.x,
				0, 1, 0, x.y,
				0, 0, 1, x.z,
				0, 0, 0, 1

			);

		} else { 
			...
		}
}
```

### CODE REVIEW

File: types/three/src/math/Matrix3.d.ts [hunk 1]
```diff
@@ -113,7 +114,7 @@ export class Matrix3 implements Matrix {
      * @param x the amount to translate in the X axis.
      * @param y the amount to translate in the Y axis.
      */
-    makeTranslation(x: number, y: number): this;
+    makeTranslation(v: Vector2): Matrix3;
     makeTranslation(x: number, y: number): Matrix3;
```
Comment (Methuselah96) [inline types/three/src/math/Matrix3.d.ts:118]: Can you make the return type `this` for both methods?

File: types/three/src/math/Matrix4.d.ts [hunk 2]
```diff
@@ -147,6 +147,7 @@ export class Matrix4 implements Matrix {
     /**
      * Sets this matrix as translation transform.
      */
+    makeTranslation(v: Vector3): Matrix4;
     makeTranslation(x: number, y: number, z: number): Matrix4;
```
Comment (Methuselah96) [inline types/three/src/math/Matrix4.d.ts:151]: Can you make the return type `this` for both methods?

**Review by Methuselah96 — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Comment by Methuselah96** [comment]
I also just realized this is part of the r153 release so the base branch should be `dev` instead of `master`. If you can't get around to these changes within the next few days, I can update the PR myself.

**Comment by puxiao** [comment]
Sorry, i resubmitted.
https://github.com/three-types/three-ts-types/pull/488

**Comment by Methuselah96** [comment]
No problem, thanks!

**Pull request closed without merging**